requirements.txt

```
@@ -2,7 +2,7 @@
 # See: https://github.com/Azure/azure-cli/issues/16858
 cryptography==3.4.7
 PyYAML==5.4.1
-awscli==1.20.1
+awscli==1.20.10
```

There is a breaking change in [1.20.9](https://github.com/aws/aws-cli/blob/develop/CHANGELOG.rst#1209) : `aws eks get-token` returns a different API version.
```suggestion
awscli==1.20.8
```

---

Breaking change reverted in v2.2.25. Expected to be reverted in 1.20.12.
See https://github.com/aws/aws-cli/issues/6308